galactic/distribution.yaml

```diff
@@ -3812,6 +3812,22 @@ repositories:
       url: https://github.com/KumarRobotics/ublox.git
       version: foxy-devel
     status: maintained
+  ublox_dgnss:
+    release:
+      packages:
+      - ublox_dgnss
+      - ublox_dgnss_node
+      - ublox_ubx_interfaces
```

Comment:
I notice [this package.xml says it is GPL](https://github.com/aussierobots/ublox_dgnss/blob/20ba2bac3b005beaaa0378aaae0fb7929ec283d1/ublox_ubx_interfaces/package.xml#L9), but the [LICENSE file says Apache 2.0](https://github.com/aussierobots/ublox_dgnss/blob/main/LICENSE). Please either: 

* create a `LICENSE` file inside the `ublox_ubx_interfaces` folder with the GPL License text

OR

* If the `package.xml` is incorrect, please change the `<license>` tag inside it to say Apache-2.0 and re-bloom the package.

Reply:
have corrected and re-bloom